test/integration_test.js

```diff
@@ -78,19 +72,42 @@ describe('Integration test on ClientService', async () => {
     it('should works as expected when executing a registered Contract',
         async () => {
           const response = await clientService.executeContract(mockedContractId,
-              mockedContractArgument, functionArgument);
+              mockedContractArgument, {});
           const responseArgumentObj = JSON.parse(response.array[2]);
           assert.equal(responseArgumentObj.asset_id, mockedAssetId);
           assert.equal(responseArgumentObj.state, mockedState);
-          assert.equal(responseArgumentObj.properties, property.properties);
+          assert.equal(responseArgumentObj.properties,
+              contractProperty.properties);
           assert.equal(response.getStatus(), 200);
         });
+    const mockedFunctionArgument = {
+      asset_id: mockedAssetId,
+      state: mockedState,
+    };
     it('should works as expected when executing a registered Function',
         async () => {
+          const contractArgumentWithFunction = {
+            asset_id: mockedAssetId,
+            state: Date.now(),
+            _functions_: [mockedFunctionId],
+          };
           const response = await clientService.executeContract(
-              mockedContractId, mockedFunctionArgument, functionArgument);
+              mockedContractId,
+              contractArgumentWithFunction,
+              mockedFunctionArgument,
+          );
           assert.equal(response.getStatus(), 200);
         });
+    it('should return a proper value on cassandra query', async () => {
+      cassandraClient = new cassandra.Client({
+        contactPoints: ['127.0.0.1:9042'],
+        localDataCenter: 'datacenter1',
+      });
+      const cassandraRespond = await cassandraClient.execute(
+          'SELECT * FROM foo.bar');
+      chai.expect(cassandraRespond.rows[0].column_a).to.be.a('string');
+      await cassandraClient.shutdown();
```

Comment:
Let's move this code block inside the `` it('should works as expected when executing a registered Function',`` test as this block is only checking Cassandra and not running a contract/udf.